stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.18.1
+version: 0.18.2
```

Comment:
I'd say 0.19.0 since it is a new feature (and people could notice the param is now no longer in use and can remove it).

Reply:
Ahh, good call. Pushing the change now.